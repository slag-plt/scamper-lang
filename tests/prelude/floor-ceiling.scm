(floor 0.71)
(floor 111)
(floor 6.1)
(floor 0.69)
(floor 0.10000000000000009)
(floor 0)
(ceiling 0.71)
(ceiling 111)
(ceiling 6.1)
(ceiling 0.69)
(ceiling 0.10000000000000009)
(ceiling 0)