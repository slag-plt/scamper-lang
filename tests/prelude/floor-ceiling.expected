0
111
6
0
0
0
1
111
7
1
1
0
